mesonbuild/minstall.py

```diff
@@ -161,8 +161,11 @@ def chown(path: T.Union[int, str, 'os.PathLike[str]', bytes, 'os.PathLike[bytes]
         real_os_chown(path, uid, gid, dir_fd=dir_fd, follow_symlinks=follow_symlinks)
 
     try:
-        os.chown = chown
-        shutil.chown(path, user, group)
+        if sys.version_info >= (3, 13):
```

Comment:
Since you already are in a `try` block, why not simply try the new syntax, and `except` the `AttributeError` for the fallback version?

Reply:
You raise a good point, but perhaps not the one you meant.

abc9eb0db1197b968d0f5c6239fb170906757419

Reply:
try/except is impenetrable to tools that try to upgrade your code when you bump the minimum required python version, and you end up needing TODO comments anyway instead of self-documenting code. It's not even obvious to me that try/except is cleaner.